### PR TITLE
Fix some return docblocks.

### PR DESCRIPTION
**Pull request by lucasmichot**
Fix some return docblocks.

---

Note:
Tests and Style are fixed in #183 and #184

### CODE REVIEW

File: src/OAuth1/Server.php [hunk 1]
```diff
@@ -71,7 +71,7 @@ public function getTokenCredentials(TemporaryCredentials $temporaryCredentials,
                 $response = $client->post($uri, $headers, $bodyParameters)->send();
             }
         } catch (BadResponseException $e) {
-            return $this->handleTokenCredentialsBadResponse($e);
+            $this->handleTokenCredentialsBadResponse($e);
```
Comment (atymic) [inline src/OAuth1/Server.php:74]: this seems like a functional change?

Comment (lucasmichot) [reply src/OAuth1/Server.php:74]: It does not, `handleTokenCredentialsBadResponse` does not return anything (`void`), it only throws an exception.

**Comment by atymic** [comment]
Could you fix the conflicts here please :)

**Comment by lucasmichot** [comment]
> Could you fix the conflicts here please :)

Done @atymic

**Pull request merged**